plugin/pkg/admission/broker/authsarcheck/admission.go

```diff
@@ -66,7 +66,7 @@ func convertToSARExtra(extra map[string][]string) map[string]authorizationapi.Ex
 	return ret
 }
 
-func (s *sarcheck) Admit(a admission.Attributes) error {
+func (s *sarcheck) Admit(a admission.Attributes, o admission.ObjectInterfaces) error {
```

Comment:
this was added to the interface, not really sure what we're supposed to do with it...

Reply:
something to do for CRDs https://github.com/kubernetes/kubernetes/pull/74154
 feel free to browse through that.